internal/access/grant.go

```diff
@@ -68,6 +68,10 @@ func ListGrants(c *gin.Context, subject uid.PolymorphicID, resource string, priv
 		selectors = append(selectors, data.ByOptionalSubject(subject))
 	}
 
+	if !showSystem {
+		selectors = append(selectors, data.NotPrivilege(models.InfraConnectorRole))
```

Comment:
I think we will have to be more specific than this.

There could be a privilege called "connector" in other places outside of infra that its legit to create a grant for. We should hide system grants by something like `privilege != connector && destination != infra` together